README.md

```diff
@@ -187,41 +187,37 @@ apolloClient.query(
 
 ## Custom Scalar Types
 
-Apollo supports Custom Scalar Types like `DateTime` for an example.
+Apollo supports Custom Scalar Types like `Date`.
 
-You first need to define the mapping in your build.gradle file. This will tell the compiler what type to use when generating the classes.
+You first need to define the mapping in your build.gradle file. This will tell the annotation processor what type to use when generating the classes.
```

Comment:
Hm, this can be misleading as we don't really have annotation processor but rather then code generator or compiler.